### DISCUSSION TITLE
Brackets local server or for enterprises

### DISCUSSION THREAD

**Discussion by cleanet** [brackets-cont/brackets#259] [Ideas]
Hi. I have a idea for Brackets Team. I would love to that the brackets editor would has the possibility of have my self-server of brackets. Where via web I can conect me remotely in my local server and so have all my development centralised.

SImiliar to: Visual Studio Code Server https://code.visualstudio.com/docs/remote/vscode-server.

P.D: The Visual Studio Code hasn't possibility of connection multi-user. Only a user.

I think that this would win reputation no only for personal usages, else also would can be very useful for enterprises...

Please for personal usage I would like that would was free (and for enterprises with rate)

**Comment by abose** [comment]
The support for remote development workflows is planned for https://github.com/phcode-dev/phoenix after new native builds are released.
